src/views/Leads/index.js

```diff
@@ -307,6 +371,25 @@ export default class Leads extends React.PureComponent {
 
     // UI
 
+    onGridEndReached = () => {
+        const { activePage, leadsPerPage, totalLeadsCount } = this.props;
+        if (activePage === Math.ceil(totalLeadsCount / leadsPerPage)) {
+            return;
+        }
+        this.props.setLeadPageActivePage({ activePage: activePage + 1 });
+    }
+
+    getSortDetails = () => {
+        const { activeSort } = this.props;
+
+        if (!activeSort) {
+            return [iconNames.chevronDown, ''];
+        } else if (activeSort.slice(0, 1) === '-') {
+            return [iconNames.chevronDown, activeSort.slice(1)];
+        }
+        return [iconNames.chevronUp, activeSort];
```

Comment:
Why is the return value a list?
A map would be more understandable and easier to change later.